### PR TITLE
added lesson on aggregation 

### PR DESCRIPTION
**Pull request by parkerfoshay**
Added aggregation

### CODE REVIEW

File: posts/take-home.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+2. Once connected, type `use sample_restaurants` into the command line.
+
+3. Once connected to the `sample_restaurants` database copy and paste the query into the command line.
+
+## 🎂 Expected Results
+
+```sh
+{ "_id" : "Five Star Fish & Chips", "avgScore" : 31 }
+{ "_id" : "Lucky 13 Saloon", "avgScore" : 31.833333333333332 }
```
Comment (sevans924) [inline posts/take-home.md:28]:
```suggestion
{ "_id" : "Lucky 13 Saloon", "avgScore" : 31.833333332 }
```

Comment (nielsenjared) [reply posts/take-home.md:28]: Good catch!

**Review by sevans924 — APPROVED** [review]
One quick change, otherwise LGTM

**Comment by nielsenjared** [comment]
LGTM!

**Review by nielsenjared — APPROVED** [review]

**Pull request merged**